apps/network/tests/database/test_user.py

```diff
@@ -0,0 +1,13 @@
+import pytest
+
+from src.users.user import User
+from .presets.user import user_metrics
+
+
+@pytest.mark.parametrize(
+    ("email, hashed_password, salt, role"), user_metrics,
+)
+def test_create_role_object(email, hashed_password, salt, role, database):
```

Comment:
I think your function name is wrong. `test_create_user_object` is likely what you intended.